### PR TITLE
adding Avatar component

### PR DESCRIPTION
**Pull request by manojkumar8400**
some changes of general css file.

### CODE REVIEW

File: documentation/Styles/general.css [hunk 1]
```diff
@@ -29,4 +30,87 @@ ul,ol{
...
+    position: relative;
+    padding: 20px;
+    font-size: 22px;
+    border-radius: 0 0 15px 15px;
+    width: 100%;
+    position: fixed;
+    z-index: 1;
+}
+.nav-header{
+    font-size: 30px;
```
Comment (srishti-maurya) [inline documentation/Styles/general.css:51]: suggestion: you may use rem here

**Review by srishti-maurya — APPROVED** [review]

**Review by kiranpratyush — APPROVED** [review]
## in General.css
In the case of **font-size** I think you should not use pixel values. correct me If I am wrong.
Else looks fine to me

**Comment by tanveertkd** [comment]
You might want to add some margin to your document body. The navigation bar is overlapping the heading.

**Review by tanveertkd — APPROVED** [review]

File: documentation/Styles/general.css (hunk 1 above)
Comment (manojkumar8400) [reply documentation/Styles/general.css:51]: thank you I would definitely go through your suggestion.

**Comment by manojkumar8400** [comment]
Thank you guys to give valuable reviews and suggestions.

**Pull request merged**